### PR TITLE
rustfmt on assets crate

### PR DESCRIPTION
**Pull request by Rhuagh**

### CODE REVIEW

**Comment by Xaeroxe** [comment]
@torkleyy please merge if you approve

**Comment by Rhuagh** [comment]
Updated imports based on @torkleyy's feedback.

File: amethyst_assets/examples/asset.rs [hunk 1]
```diff
@@ -7,9 +7,10 @@ extern crate rayon;
 use std::str::{Utf8Error, from_utf8};
 use std::sync::Arc;
 
-use amethyst_assets::*;
 use rayon::{Configuration, ThreadPool};
 
+use amethyst_assets::*;
```
Comment (torkleyy) [inline amethyst_assets/examples/asset.rs]: From the example's point of view, I don't think it's local

**Comment by Rhuagh** [comment]
Import order should be fixed now.

**Pull request merged**